### ISSUE TITLE
Automatic Version Bump

### ISSUE THREAD

**Issue by mercuriete** [mercuriete/android-musician-tools#29]
As a < type of user >, I want < some goal > so that < some reason >.
Fuking Scrum

I as a product_owner/analyst/developer/tester/user, I want to do automatic version bump so that i can do automatic version bump.

----

OK but now serious

I want to implement automatic version bump because I dont want to make a commit everytime I need to do a release on Google Play.

We would like to follow semver but with semver is imposible to do automatic version bump so:

the app version will follow the following schema:

versionName => year.month.day

versionCode => number of commits from initial commit 
```bash
 git rev-list --count HEAD
```

With this changes I can deploy to Google Play a consistent version number but a different versionName per day.
This will have some pros and some cons.
It will fix the problem of having to hardcode the version and make a commit each time we want do do a release, but the problem is we cant do a version without fooling the system clock.

TLDR;
I don't want trash commits hardcoding version numbers.
So I want to automate versionCode and versionName.

Edit: the motivation of changing to a year.month.day schema is some projects in the wild are changing to that schema and I want to follow the convention.
